### DISCUSSION TITLE
How to show only unread notifications

### DISCUSSION THREAD

**Discussion by maan2003** [magit/forge#341] [Support] [answered]
I can't find a way to show only unread notifications

**Comment by tarsius** [comment, marked as answer]
The whole notification thing would need an overhaul, but it is not priority for a variety of reasons.

Meanwhile you can do this:

```elisp
(with-eval-after-load 'magit
  (fset 'forge--list-notifications-all
        (symbol-function 'forge--list-notifications-unread)))
```

To always list only the unread notifications, which might be what you want in the short-run because listing everything is so extremely slow.

**Reply by zoechi** [reply]
I get an error void-function for `forge--list-notifications-all` and this was the only search result where this function is mentioned. What could I be doing wrong?

**Reply by zoechi** [reply]
Ok, it was renamed and the error came from gh-notify. There is already a PR
